sync/tasks.py

```diff
@@ -47,9 +46,8 @@ def inner(*args, **kwargs):
             # run before this task completes. That means that manual commands
             # will always  win over celery tasks, which we want
             time.sleep(0.1)
-        except Exception as e:
-            logger.error(str(six.text_type(e).encode("utf8")))
-            logger.error("".join(traceback.format_exc(e)))
+        except Exception:
+            logger.error("".join(traceback.format_exc()))
```

Comment:
for myself [`traceback.format_exception_only(etype, value)`](https://docs.python.org/3/library/traceback.html#traceback.format_exception_only)